lib/src/main/kotlin/com/lemonappdev/konsist/core/provider/KoFullyQualifiedNameProviderCore.kt

```diff
@@ -16,7 +16,7 @@ internal interface KoFullyQualifiedNameProviderCore :
             var fqn = containingFile
                 .imports
                 .map { it.name }
-                .firstOrNull { it.contains(textUsedToFqn) }
+                .firstOrNull { it.endsWith(".$textUsedToFqn") }
 
```

Comment:
Shouldn't a similar treatment be applied line 32 when going through the declarations in the file, in case a declaration in the file contains as substring the textUsedToFqn ? (example if the file contains the declaration of class PixelSize and class Pixel) ? 

Instead of using `it.endsWith(".$textUsedToFqn")`, it is the equality of the string that needs to be compared